src/plugins/template/tests/functional/op_reference/tile.cpp

```diff
@@ -104,11 +104,11 @@ std::vector<TileParams> generateParams() {
                    "tile_3d_to_3d_repeats_broadcast"),
         TileParams(reference_tests::Tensor(ET, {1}, std::vector<T>{1}),
                    reference_tests::Tensor(ET_INT, {3}, std::vector<T_INT>{0, 2, 3}),
-                   reference_tests::Tensor(ET, {0}, std::vector<T>{}),
+                   reference_tests::Tensor(ET, {0, 2, 3}, std::vector<T>{}),
                    "tile_1d_to_3d_with_zero_on_axis_0"),
         TileParams(reference_tests::Tensor(ET, {3}, std::vector<T>{1, 2, 3}),
                    reference_tests::Tensor(ET_INT, {3}, std::vector<T_INT>{2, 0, 3}),
-                   reference_tests::Tensor(ET, {0}, std::vector<T>{}),
+                   reference_tests::Tensor(ET, {2, 0, 9}, std::vector<T>{}),
```

Comment:
Why it is required? I expected tensor with no values should have shape == {0}

Reply:
Well .. shapes comparison fails otherwise. Should the comparison be improved? If so, I'll move this test back to legacy raw compare.

Reply:
However this shape is produced by `shape_infer`, so it should be consider valid. @praasz, what do think?

Reply:
Tensor can have shape set with dimension `0` which will result as no data but still can have some dimension.

If these test validate all properties of the tensor like type, shape, data then it should be set what shape inference calculate for this operator (in this case Tile should not reduce rank.
There will be just no data to calculate/compare.

In my opinion this is OK as we check what Tile produce.

Maybe is not real scenario but Is possible to make this kind of Tile as sub-graph and gather first and/or last dimension(s) from Tile output make ShapeOf of it which can produce data like {2,9} which can be input for other operator.